__tests__/api.test.js

```diff
@@ -119,6 +119,56 @@ describe("GET/api/articles", () => {
       });
   });
 });
+describe('GET/api/articles(sort queries)',()=>{
+  it('status 200, responds with articles sorted by given property and ordered descending when order is not given', ()=>{
+    return request(app)
+    .get('/api/articles?sort_by=title')
+    .expect(200)
+    .then(({body})=>{
+      expect(body.articles).toBeSortedBy('title', {descending: true })
+    })
+  })
+  it("status 400: returns an error message when given wrong property", () => {
+    return request(app)
+      .get("/api/articles?sort_by=banana")
+      .expect(400)
+      .then(({body}) => {
+        expect(body.msg).toBe("Invalid Sort");
+      });
+  });
+  it("status 400: returns an error message when no property is given", () => {
+    return request(app)
+      .get("/api/articles?sort_by=")
+      .expect(400)
+      .then(({body}) => {
+        expect(body.msg).toBe("Invalid Sort");
+      });
```

Comment:
Dealers choice with the above test - this is fine and also you could just return the articles with the default sort and ignore it...upto you!